english/call-a-doctor/let-met-out.md

```diff
@@ -0,0 +1,9 @@
+The doctor frowns. He shakes his head and scribbles something on a page inside a folder 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.